WcaOnRails/app/models/competition.rb

```diff
@@ -29,6 +29,8 @@ class Competition < ActiveRecord::Base
            allow_nil: true,
            with_model_currency: :currency_code
 
+  scope :present, -> { where("CAST(CONCAT(endYear,'-',endMonth,'-',endDay) as Datetime) >= ?", Date.today) }
```

Comment:
I do prefer `not_ended` or the slightly less awkward `not_over` for code purposes. I'm okay continuing to use present for user facing stuff.